study.md

```diff
@@ -29,21 +29,23 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The javascript process that moves functions from the event queue to the staging area into the call stack.
 ```
 
 ## Synchronicity and Asynchronicity
 
 In your own words, describe the difference between sync and async.
 
 ```md
-<!-- your answer here -->
+Async is non-blocking i/o and sync is i/o that operates in an order that can block other functions/processes.
 ```
```

Comment:
To simplify, synchronous processes happen in order one by one, where as async doesn't!